src/app/home/home.component.scss

```diff
@@ -1,107 +1,101 @@
+@import "../_variables.scss";
 
-.greetings {
-  margin-top: 30px;
-  margin-bottom: 30px;
-}
-
-.greetings-hero-img {
-  position: center;
-  height: 80%;
-  width: 80%;
-  border-radius: 50%;
-}
-
-.mobile-greetings-hero-img {
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-  position: center;
-  height: 40%;
-  width: 40%;
-  border-radius: 50%;
-}
-
-.nomargin {
-  margin: 0;
-}
-
-.grid-container {
-  margin-top: 20px;
-  margin-bottom: 20px;
+.background {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: $black;
+  z-index: -999; // hacky way to override app background color for now
 }
 
-.dashboard-card {
-  position: absolute;
-  top: 15px;
-  bottom: 15px;
+.landing-page {
+  background: $black;
+  font-family: Roboto, sans-serif;
```

Comment:
`font-family` can be very handy to put in a variable to ensure consistent fallbacks and give a single place in the code to change them if you ever need to do that since there's so many elements that need to have this explicitly set since they can't inherit the font from the body (mainly inputs/buttons).